src/Symfony/Component/Validator/Constraints/Type.php

```diff
@@ -31,7 +31,7 @@ class Type extends Constraint
     public string|array|null $type = null;
 
     /**
-     * @param string|string[]|array<string,mixed>|null $type    The type(s) to enforce on the value
+     * @param string|string[]|array<string,mixed>|array<string,array<mixed>>|null $type    The type(s) to enforce on the value
```

Comment:
```suggestion
     * @param string|list<string>|array<string,mixed>|null $type The type(s) to enforce on the value
```

please mind aligning the other arguments

Reply:
thx for the comment, done